src/connectors/radiozenders.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+let reverse = false;
+if ($('.player-station-title').text().includes('Qmusic')) {
+	reverse = true;
+}
+
+Connector.playerSelector = '#jp_container_1';
+
+Connector.getArtistTrack = function() {
+	let artistTrack = $('#player-station-info').text();
+	return Util.splitArtistTrack(artistTrack, [' - '], reverse);
+};
+
+Connector.playButtonSelector = '.player-equalizer';
```

Comment:
I think this needs some more work. This is used to determine if the player is playing or not. Looking at the site I think we should overload the `Connector.isPlaying` function like this:

```
Connector.isPlaying = () => {
	return $('#jp_container_1').hasClass('jp-state-playing');
};
```

Reply:
Changed it, thanks